fuzzy-search.js

```diff
@@ -49,19 +51,17 @@ function fuzzySearch(haystackList, needle, renderer, caseSensitive = false) {
 
 		// Check the score of searchItem and act accordingly
 		if (score == 1) {
-			preciseMatch.push(haystackList[i]);
+			preciseMatch += `${haystackList[i]} `;
 		} else if (score == 2) {
-			preciseMatch.splice(0, 0, haystackList[i]);
```

Comment:
Replacing `splice` with `string concatenation` as splice is a much more expensive operation, as shown in below screenshot
![image](https://user-images.githubusercontent.com/3054432/104835687-40498780-58ce-11eb-9159-85adfe916a6e.png)